### DISCUSSION TITLE
tests Parte 1

### DISCUSSION THREAD

**Discussion by blampasona18** [IIC2133-PUC/2024-2#151] [Tarea 2] [answered]
Hola tengo una duda con algo que pasa en varios tests. Por ejemplo en el pegasus-tests/hard-1 se tiene lo siguiente:

- En input.txt : IN-CIRCLE 122 191 3
- En output.txt : IN-CIRCLE 122 191

Esto ocurre en los casos en que no encuentra penguins en ese rango. Mi función maneja bien para que no se imprima un '\n' pero el radio debería imprimirse igual, o al menos el enunciado no dice nada de eso.

**Comment by sebacarrascop** [comment, marked as answer]
hola!, sii, eso fue un error, hace unos dias subimos actualizamos los tests y eso está arreglado.
Descarga los nuevos tests y comentame si tienes algún problema.
